manim/mobject/svg/brace.py

```diff
@@ -134,17 +135,28 @@ def get_direction(self):
 
 
 class BraceLabel(VMobject, metaclass=ConvertToOpenGL):
+    @deprecated_params(
+        params="label_scale",
+        since="v0.10.0",
+        until="v0.11.0",
+        message="Use font_size instead.",
+    )
     def __init__(
         self,
         obj,
         text,
         brace_direction=DOWN,
         label_constructor=MathTex,
-        label_scale=1,
+        font_size=DEFAULT_FONT_SIZE,
```

Comment:
Similar to above: `font_scale` vs. `label_font_scale`?

Reply:
Same reasoning as above